src/RegisteredID.php

```diff
@@ -92,21 +94,20 @@ public function showChildForItemForm($canedit, $field_name, $id, bool $display =
         }
         $value             = htmlspecialchars($value);
         $result            = "";
-        $main_field        = $field_name . "[$id]";
-        $type_field        = $field_name . "_type[$id]";
-        $registeredIDTypes = self::getRegisteredIDTypes();
+        $main_field        = htmlspecialchars($field_name . "[$id]");
+        $type_field        = htmlspecialchars($field_name . "_type[$id]");
+        $registeredIDTypes = htmlspecialchars(self::getRegisteredIDTypes());
```

Comment:
This is an array. Each item seems to be escaped in the `foreach ($registeredIDTypes as $name => $label) {` loop.
```suggestion
        $registeredIDTypes = self::getRegisteredIDTypes();
```